webapp/src/shared/components/Input/Input.css

```diff
@@ -0,0 +1,39 @@
+.form-container {
+  width: 312px;
+}
+
+.input-container {
+  margin-top: 0.3rem;
+  display: flex;
+  padding: 0.8rem 0.6rem;
+  border-radius: 0.4rem;
+  gap: 0.4rem;
+}
+
+.input-container-dark {
+  background-color: var(--color-dark);
+}
+
+.input-container-light {
+  background-color: var(--color-light);
+}
+
+.input-form {
+  border: 0;
+  width: 100%;
+}
+
```

Comment:
maybe we would like to control the color of the placeholder text? Right now it's the default color...

Reply:
If we want to have different colors for the placeholder in the dark or light scenario, we can add something like that, sure.